pkg/component/etcd/statefulset/values.go

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) 2022 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package statefulset
+
+import (
+	druidv1alpha1 "github.com/gardener/etcd-druid/api/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+// Values contains the values necessary for creating ETCD statefulset.
+type Values struct {
+	// Name is the name of the etcd resource.
+	Name string
```

Comment:
Hm, Name is the name to be used for the `StatefulSet` and since it's in the StatefulSet component package, this is evident I think.
Unlike UID which is really the UID of the `etcd` and not the one from the StatefulSet or any other object.

Reply:
Ok then the comment needs to be corrected.